.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
```

Comment:
exist in subfolder [prototype](https://github.com/eclipse/org.eclipse.sensinact.gateway/tree/future/prototype/prototype)/.editorconfig